src/types/types.ts

```diff
@@ -1221,6 +1231,11 @@ export interface EdgeCurrencyWallet {
     opts?: EdgeSignMessageOptions
   ) => Promise<string>
 
+  readonly signBytes: (
```

Comment:
Please add an `@deprecated` comment to the old `signMessage`, and stick it at the bottom of the interface. Same for the engine.